src/patternfly/components/Form/form.scss

```diff
@@ -108,6 +122,10 @@
 
 .pf-c-form__group-label {
   padding-bottom: var(--pf-c-form__group-label--PaddingBottom);
+
+  .pf-c-form__label + .pf-c-form__helper-text {
```

Comment:
do you need this or could you just set `--pf-c-form__helper-text--MarginTop: 0;` on `.pf-c-form__group-label`?